modules/standard/DateTime.chpl

```diff
@@ -474,7 +478,7 @@ module DateTime {
     pragma "no doc"
     var chpl_hour, chpl_minute, chpl_second, chpl_microsecond: int;
     pragma "no doc"
-    var chpl_tzinfo: TZInfo;
+    var chpl_tzinfo: Shared(TZInfo);
```

Comment:
Can you point me to wherever this is set to something other than Shared(nil)?

Reply:
It's used in some of the DateTime tests: [testDatetimeTZ](https://github.com/chapel-lang/chapel/blob/master/test/library/standard/DateTime/testDatetimeTZ.chpl), [testTimezone](https://github.com/chapel-lang/chapel/blob/master/test/library/standard/DateTime/testTimezone.chpl), [testTimezoneConversions](https://github.com/chapel-lang/chapel/blob/master/test/library/standard/DateTime/testTimezoneConversions.chpl)